src/Symfony/Bundle/WebServerBundle/WebServerConfig.php

```diff
@@ -69,6 +70,8 @@ public function __construct($documentRoot, $env, $address = null, $router = null
         if (!ctype_digit($this->port)) {
             throw new \InvalidArgumentException(sprintf('Port "%s" is not valid.', $this->port));
         }
+
+        $this->executable = $executable;
```

Comment:
I welcome any suggestions on how to validate this. Using `is_executable` would only work for full paths. And this can be anything from `xdebug` (from my original issue) to `/usr/bin/php7_custom_build -c /custom/php.ini`.